src/mixins/filing-template-mixin.ts

```diff
@@ -413,9 +440,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         validationItemDetails: []
       },
       resolutionConfirmed:
-        draftFiling.dissolution?.specialResolution?.resolutionConfirmed ||
+        draftFiling?.specialResolution?.resolutionConfirmed ||
```

Comment:
You don't need conditional chaining on `draftFiling` (since no other code in the method uses it).

Reply:
this is actually req'd.  i tested creating a new coop voluntary dissolution filing and it errors out.  i've kept this in for now but I do plan on taking a look at restructuring the code in this block on the PR where I will create the resolution summary component.  

i.e this block of code
```
    // restore Resolution data
    const createResolution: CreateResolutionIF = {
      validationDetail: {
        valid: false,
        validationItemDetails: []
      },
      resolutionConfirmed:
        draftFiling?.specialResolution?.resolutionConfirmed ||
        draftFiling.dissolution?.resolution?.resolutionConfirmed || false
    }
    if (draftFiling.specialResolution) {
      createResolution.resolutionDate = draftFiling.specialResolution.resolutionDate
      createResolution.resolutionText = draftFiling.specialResolution.resolution
      createResolution.signingPerson = draftFiling.specialResolution.signatory
      createResolution.signingDate = draftFiling.specialResolution.signingDate
    }
    this.setResolution(createResolution)
```

Reply:
I'm referring to `draftFiling?` ... tons of other code in this method dereferences it, so I think it's safe to assume `draftFiling` exists.

Reply:
oh ic.  k will fix that up in subsequent work.